.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```

Comment:
I don't think screenshots apply to docmaker as any PDF rendering issues are caused by user error or errors in Weasyprint.

Reply:
Screenshots of terminal output could apply

Reply:
I wouldn't create a separate section for that then

Reply:
Removed section and (sortof) merged it with the additional context header